packages/zk-wizard/src/zk-wallet-account.ts

```diff
@@ -101,7 +101,9 @@ export class ZkWalletAccount {
   addERC20(...addresses: string[]) {
     this.erc20.push(
       ...addresses
-        .filter(addr => this.erc20.find(Address.from(addr).eq) === undefined)
+        .filter(
+          addr => this.erc20.find(e => Address.from(addr).eq(e)) === undefined,
```

Comment:
This is basically same as the current code, but it as throwing below error when the input addresses was string.

```
TypeError: Cannot read properties of undefined (reading 'toBN')
```
This is probably because of usage of `this` inside the soltypes library and passing the function as is to `find` is changing the scope of `this`. I didn't debug the exact place, but can confirm this change works as expected.